### ISSUE TITLE
Angular: Add FAQ

### ISSUE THREAD

**Issue by rozzan** [zeldaret/website#30]
We need an FAQ displayed to help new users get a grasp on what is going on. Previously, there has been discussion about a potential mini FAQ on the landing page, and a separate larger FAQ. Unsure whether this would be a site-wide FAQ, or if it would be split by game - this should be decided first.

**Acceptance criteria**
* Discuss and agree on the scope of the FAQ
* Implement appropriate components

**Comment by EllipticEllipsis** [comment]
Draft of 100% mini-FAQ:

# Ocarina of Time Decomp 100% FAQ

## What is this project's goal?

We aim to produce a complete documented codebase, written in C as far as possible, for every version of The Legend of Zelda Ocarina of Time. This may be used for knowledge of the game (how its systems work, for glitches, for speedruns, etc.), or for changing parts of the game to make it act differently or do different things (that is, modding). This code compiles to be byte-for-byte identical to the original ROM, but our code almost certainly looks very different from the original, since it is based only on analysing the compiled code.


## What does 100% mean?

100% means that, in the first version of the game we have worked on, all of the code that can be written as compilable C code has been. It does **not** mean that the code is in any sense ready to be used easily for mods (including any ports). It also does not mean that the code is documented: there is still *a lot* of work to do on that.


## I heard you're at 100%! Why does the graph not show 100%?

The graph indicates the status of the main repository. The last decompiled code is on a contributor's local copy while they clean it up and prepare it for review. It will then be reviewed by contributors, and only after that will it be merged into the main repository and the graph will show 100%.


## PC port?

The goal of this project is **NOT** to make any kind of port, such as to PC. Someone else might take our work and make a port (although they would be foolish to do so before the codebase is improved) but **ZeldaRET will not be involved in any porting**. A port will not happen tomorrow anyway: decomp would only be the first step.


## What next?

- Reorganisation of systems
- Restructuring of headers
- Documentation
- Other versions: 17 of them!

## Where can I find out more information?

- You can consult the [main FAQ](faq.html)
- You can join the [ZeldaRET Discord server](http://discord.zelda64.dev/)
- If you want to write an article or make a video about decomp, please reach out to one of the project leads via Discord: they are happy to explain any aspects of the project you would like to know about.
